### PR TITLE
add tdd CommentCount component

### PR DESCRIPTION
**Pull request by sowasred2012**

### CODE REVIEW

File: components/CommentCount/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    return (
+      <div>
+        {this.formatString(this.props.count)}
+      </div>
+    )
+  }
+}
+
+CommentCount.defaultProps = {
+  count: 0
```
Comment (adrianclay) [inline components/CommentCount/index.js:22]: Interesting, good to know you can specify defaults.

File: components/CommentCount/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      return `${count} comment`
+    } else {
+      return `${count} comments`
+    }
+  }
+
+  render () {
+    return (
+      <div>
+        {this.formatString(this.props.count)}
```
Comment (adrianclay) [inline components/CommentCount/index.js:15]: I quite liked @camillavk naming where she was more specific about what was being formatted.

https://github.com/madetech/react-learning-plan/pull/14/files#diff-2d60ad83073fa41956d6885ddccb87d6R13

File: components/CommentCount/index.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+class CommentCount extends React.Component {
+  formatString(count) {
+    if(count === 1) {
+      return `${count} comment`
```
Comment (adrianclay) [inline components/CommentCount/index.js:6]: Feels as though we are reimplementing gettext plurals, https://www.gnu.org/savannah-checkouts/gnu/gettext/manual/html_node/Plural-forms.html

I'd imagine there is a similar library for JS.

Comment (camillavk) [reply components/CommentCount/index.js:6]: I originally looked for a library when doing mine but as it was such a small implementation I thought it was more work to implement the library than to not implement it TBH

**Pull request closed without merging**